src/QueryPipeline/QueryPipelineBuilder.h

```diff
@@ -159,6 +159,7 @@ class QueryPipelineBuilder
     /// Convert query pipeline to pipe.
     static Pipe getPipe(QueryPipelineBuilder pipeline) { return std::move(pipeline.pipe); }
     static QueryPipeline getPipeline(QueryPipelineBuilder builder);
+    static void initRowsBeforeLimit(IOutputFormat * output_format);
```

Comment:
The non static declaration can be deleted now right?

Reply:
What do you mean by "non static declaration" ?

Reply:
Ah, I see.